### PR TITLE
Feature/add asg bid leg

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

File: src/Components/Agenda/AgendaItemMaintenanceContainer/AgendaItemMaintenanceContainer.jsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ const AgendaItemMaintenanceContainer = (props) => {
   const [agendaItemTimelineLoading, setAgendaItemTimelineLoading] = useState(true);
   const [legs, setLegs] = useState([]);
   const [maintenanceInfo, setMaintenanceInfo] = useState([]);
+  const [asgSepBid, setAsgSepBid] = useState({});
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemMaintenanceContainer/AgendaItemMaintenanceContainer.jsx:27]: Can remove this - will just need to refactor this variable out

File: src/Components/Agenda/AgendaItemMaintenancePane/AgendaItemMaintenancePane.jsx [hunk 2]
```diff
@@ -48,7 +48,6 @@ const AgendaItemMaintenancePane = (props) => {
   const panelDatesML = filter(panelDates, (p) => p.pmt_code === 'ML');
   const panelDatesID = filter(panelDates, (p) => p.pmt_code === 'ID');
 
-  const [asgSepBid, setAsgSepBid] = useState('');
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemMaintenancePane/AgendaItemMaintenancePane.jsx]: Wow - why would you remove this 😼

Comment (elizabeth-jimenez) [reply src/Components/Agenda/AgendaItemMaintenancePane/AgendaItemMaintenancePane.jsx]: 😹

**Review by scott062 — APPROVED** [review]
🦿 🦵 🍗

**Pull request merged**